elide-async/pom.xml

```diff
@@ -113,6 +113,11 @@
             <version>3.0.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20190722</version>
```

Comment:
There is a new 2020 version lets use that.

Reply:
changed it to 2020 version